tools/codeowners-utils/Azure.Sdk.Tools.CodeownersUtils/Constants/ErrorMessageConstants.cs

```diff
@@ -23,6 +23,8 @@ public class ErrorMessageConstants
         public const string InvalidTeamPartial = " is an invalid team. Ensure the team exists and has write permissions.";
         public const string InvalidUserPartial = " is an invalid user. Ensure the user exists, is public member of Azure and has write permissions.";
         public const string MalformedTeamEntryPartial = " is a malformed team entry and should start with '@Azure/'.";
+        public const string PathEntryMissingOwnersPartialStart = "Path entry, ";
```

Comment:
Any reason to not just have one string with a place holder? `Path entry, {0}, is missing owners.`

Reply:
> Any reason to not just have one string with a place holder? `Path entry, {0}, is missing owners.`
I wasn't aware this actually worked. I've updated the const. The only unfortunate thing here is that the TestCase declaration complains about `string.Format(ErrorMessageConstants.PathEntryMissingOwners, line.Trim())`  and requires the constant string. That's minor all things considered.